tock/utilization/org.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
```

Comment:
I assume that when we get the business units into a db table, we'll have a lot easier time with them, right?